aspnet/security/cors/sample/src/CorsExample2/Startup.cs

```diff
@@ -1,6 +1,6 @@
 ﻿using Microsoft.AspNet.Builder;
```

Comment:
All the samples here seem to use Cors middleware only...this is not very common thing to do as far as I know...Cors can be used without the CorsMiddleware...i.e MVC middleware has support for Cors (which it does through the Cors authorization filter)...I think we should have more samples using this...